<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>421</width>
      <height>401</height>
     </rect>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_word">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>10</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_suspend">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Suspend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kill">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kill</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>90</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resume">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>90</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mutex</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Critical section</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_priority_num">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>390</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_kill_num">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>310</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_resume_num">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>210</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_suspend_num">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>160</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_priority">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>390</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>IDLE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LOWEST</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BELOW NORMAL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NORMAL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ABOVE NORMAL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HIGHEST</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_change_priority">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change priority</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>460</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>901</width>
      <height>85</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
